README.md

```diff
@@ -1 +1,92 @@
-# covid-19-tracker
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# COVID-19 TRACKER APP
+
+> This is a web app that display metric data from an API for covid-19 cases in different countries.
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- Javascript, HTML, CSS
+- React
+- Redux
+- API
+
+## Live Demo
+Click here to visit a [Live Demo Page](https://covid-19-tracker-kwentsir.netlify.app/) 
+
+
+## Project Description
+
+Click here to watch a [video](https://www.loom.com/share/182fe0fd0fc441338565318926d18a31) that gives a brief description of the project
+
+## Getting Started with Create React App
+
+**Installation**
+
+- Clone the repository
+
+```
+git clone git@github.com:Kwentsir/covid-19-tracker.git
+
+cd covid-19-tracker
+```
+
+- Install NPM packages
+
+```
+npm install
+```
+
+- Run server
+
+```
+npm start
+```
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+## Author
+
+👤 **Joseph Kwesi Kwentsir**
+
+- GitHub: [@Kwentsir](https://github.com/Kwentsir/)
+- Twitter: [@Kwentsir](https://twitter.com/jkwentsir)
+- LinkedIn: [Joseph Kwesi Kwentsir](https://www.linkedin.com/in/josephkwentsir/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- UI Inspired by [Nelson Sakwa](<https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)>) on Behance
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] Kindly add your MIT license to your project. I suggest you add some screenshots from your project to your Readme file.